### ISSUE TITLE
Uncaught Error: Must pass a styleManager parameter when initializing a StylesElement

### ISSUE THREAD

**Issue by KCErb** [Galadirith/markdown-preview-plus#155]
Steps to reproduce:

1. Markdown file with images and latex equations
2. In preview mode select "save as HTML

**Atom Version**: 1.2.4
**System**: Mac OS X 10.11.1
**Thrown From**: [markdown-preview-plus](https://github.com/Galadirith/markdown-preview-plus) package, v2.2.2

### Stack Trace

Uncaught Error: Must pass a styleManager parameter when initializing a StylesElement

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/styles-element.js:63

Error: Must pass a styleManager parameter when initializing a StylesElement
    at atom-styles.StylesElement.initialize (/Applications/Atom.app/Contents/Resources/app.asar/src/styles-element.js:63:15)
    at MarkdownPreviewView.module.exports.MarkdownPreviewView.getTextEditorStyles (/Users/KC/.atom/packages/markdown-preview-plus/lib/markdown-preview-view.coffee:228:22)
    at MarkdownPreviewView.module.exports.MarkdownPreviewView.getMarkdownPreviewCSS (/Users/KC/.atom/packages/markdown-preview-plus/lib/markdown-preview-view.coffee:246:16)
    at /Users/KC/.atom/packages/markdown-preview-plus/lib/markdown-preview-view.coffee:328:15
    at /Users/KC/.atom/packages/markdown-preview-plus/lib/renderer.coffee:32:5
    at Function.callbackFunction (/Users/KC/.atom/packages/markdown-preview-plus/lib/renderer.coffee:43:5)
    at handleResponse (/Users/KC/.atom/packages/markdown-preview-plus/lib/pandoc-helper.coffee:131:19)
    at ChildProcess.<anonymous> (/Users/KC/.atom/packages/markdown-preview-plus/node_modules/pdc/pdc.js:65:5)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:817:16)
    at Socket.<anonymous> (internal/child_process.js:319:11)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at Pipe._onclose (net.js:469:12)
```

### Commands

```
     -8:07.6.0 core:save-as (h3#q1.1.)
     -7:43.5.0 core:copy (img.)
     -7:39.5.0 application:new-file (atom-text-editor.editor.is-focused)
     -7:39 core:paste (atom-text-editor.editor.is-focused)
     -7:37 core:save (atom-text-editor.editor.is-focused)
  2x -7:29.2.0 core:close (div.markdown-preview.native-key-bindings)
     -6:58.6.0 markdown-preview-plus:toggle (atom-text-editor.editor.is-focused)
     -6:53.7.0 core:save-as (h1#homework-5.)
     -2:56.2.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-pen-paper-coffee-syntax.theme-one-light-ui)
     -0:03.6.0 core:save-as (p.)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "markdown-preview",
      "language-gfm"
    ],
    "themes": [
      "one-light-ui",
      "pen-paper-coffee-syntax"
    ]
  },
  "markdown-preview-plus": {
    "enableLatexRenderingByDefault": true,
    "breakOnSingleNewline": true,
    "pandocPath": "/usr/local/bin/pandoc",
    "enablePandoc": true
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-paths, v1.0.2
autocomplete-ruby, v0.1.0
color-picker, v2.0.13
file-icons, v1.6.12
highlight-selected, v0.11.1
language-crystal-actual, v0.28.2
language-haml, v0.23.7
language-latex, v0.6.1
language-matlab, v0.1.1
language-pfm, v0.82.0
linter, v1.11.2
linter-crystal, v3.2.3
linter-haml, v1.0.0
markdown-preview-plus, v2.2.2
mathjax-wrapper, v0.1.0
minimap, v4.16.2
minimap-highlight-selected, v4.3.1
pen-paper-coffee-syntax, v0.11.0
project-manager, v2.6.5
theme-switcher, v1.1.0
wordcount, v2.4.1

# Dev
No dev packages
```

**Comment by KCErb** [comment]
This is thrown with #143 for me. Must be related. I'm looking into it now. I'll at least try to offer more insight into why it's broken if I can't actually create a PR.

**Comment by Galadirith** [comment]
@KCErb Thanks so much for reporting this and really sorry I have been away for so long. This does look like it has the same root cause as #143. I have a candidate fix based on atom/markdown-preview#316 which I will roll out soon.

I will close this in favour of #143. Thanks again @KCErb

**Comment by KCErb** [comment]
No worries. Thanks for all of your work on this project!
